nemo/collections/asr/modules/ssl_modules/augmentation.py

```diff
@@ -232,21 +224,15 @@ def __call__(self, batch: AudioNoiseBatch) -> AudioNoiseBatch:
 
             # get the residual signal to be added to original audio
             noise_signal = mix_scale * noise_signal
-
-            # add noise to audio
-            noisy_audio[i] = audio_signal[i] + noise_signal
-            noisy_audio_len[i] = audio_lengths[i]
             noise[i] = noise_signal
             noise_len[i] = audio_lengths[i]
 
         return AudioNoiseBatch(
             sample_id=batch.sample_id,
             audio=batch.audio,
             audio_len=batch.audio_len,
-            noise=noise,
```

Comment:
We want to keep noise and noise_len so that the caller can have full awareness of what noise is applied to the original audio, in case the noise is not simply added to the original audio.